Makefile

```diff
@@ -38,6 +38,9 @@ ifndef NOBANNER
 endif
 	go install $(VT_GO_PARALLEL) -ldflags "$(tools/build_version_flags.sh)" ./go/...
 
+parser:
+	goyacc -o ./go/vt/sqlparser/sql.go -v ./go/vt/sqlparser/sql.output ./go/vt/sqlparser/sql.y
```

Comment:
We already have a Makefile in go/vt/sqlparser.

Instead of duplicating the code, can you please call that file?

The following should do that:

`make -C go/vt/sqlparser`